### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lavanyagaur22**
### Description
update readme.md by adding the voyager server url.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,5 +13,6 @@ fully featured Offline Workflow and Conflict Resolution.
 
 ## Example application
 
-See `sample` for example application
+See `sample` for example application. It uses voyager server as its backened (https://github.com/aerogear/voyager-server)
```
Comment (wtrocki) [inline README.md]: You ment ionic showcase server?

**Review by wtrocki — CHANGES_REQUESTED** [review]
small fix

File: README.md (hunk 1 above)
Comment (Lavanyagaur22) [reply README.md]: @wtrocki yeah, ionic showcase is based on voyager server so I added the link for the same

Comment (wtrocki) [reply README.md]: I mean that we need to include server that is in showcase to even run the application. This information is missing. There is no need to use voyager-sever. Voyager-server provides additional capabilities but things will work with apollo graphql as well.

**Comment by Lavanyagaur22** [comment]
@wtrocki Please have a look at this :)

File: README.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ fully featured Offline Workflow and Conflict Resolution.
...
 
-See `sample` for example application. It uses voyager server as its backened (https://github.com/aerogear/voyager-server)
+See `sample` for example application. For now it's using ionic showcase as its backened (https://github.com/aerogear/ionic-showcase)
+
+## How to run the application
+
+- Clone the ionic showcase repository (https://github.com/aerogear/ionic-showcase.git)
+- Run the following commands **to start the server:**
+ 
+  [![Screenshot-2019-06-07-at-11-14-16-PM.png](https://i.postimg.cc/28WFnK0c/Screenshot-2019-06-07-at-11-14-16-PM.png)](https://postimg.cc/N5gHcpqm)
```
Comment (wtrocki) [inline README.md]: Antipattern. Never use screenshots for commands

**Review by wtrocki — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Lavanyagaur22) [reply README.md]: Removed screenshots.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Some issues on build. Please check

**Pull request merged**